core/core-impl/src/test/java/io/fabric8/launcher/core/impl/MissionControlIT.java

```diff
@@ -64,7 +64,7 @@
     private OpenShiftServiceFactory openShiftServiceFactory;
 
     @Inject
-    private GitHubServiceFactory gitHubServiceFactory;
+    private GitServiceFactory gitServiceFactory;
```

Comment:
I'm curious: Does this test pass? AFAIK doesn't have any produces for GitServiceFactory